vlasovsolver/vlasovmover.cpp

```diff
@@ -388,12 +388,6 @@ void calculateAcceleration(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>&
            }       
            // Compute global maximum for number of subcycles
            MPI_Allreduce(&maxSubcycles, &globalMaxSubcycles, 1, MPI_INT, MPI_MAX, MPI_COMM_WORLD);
-           if (myRank == MASTER_RANK) {
-               if (globalMaxSubcycles > 10000) {
-                   logFile << "(MAIN) ERROR: Variable 'globalMaxSubcycles' above critical value! Aborting!" << endl;
-                   exit(1);
-               }
-           }
```

Comment:
I actually like this as it would kill a job where suddenly B went crazy and we "hang" in a zillion subcycles. But it's not happening that often. In any case there's no graceful bailout with a restart possible so it might as well just exit here, no?

Reply:
This is acc subcycles which hardcapped in the cfg file to 1 or 2. And it would go up only if B went WAAAAY up. This was a relic form old testing in dealing differently with electron substepping allowing some regions to go to unphysical subcycling, but since going to plasma period limited acceleration, this is no longer a valid concern.